src/icons/ArrowIcon.js

```diff
@@ -4,7 +4,7 @@ import { Icon } from '../components/Icon'
 
 const ArrowIcon = (props) => (
   <Icon { ...props }>
-    <g transform='translate(0, -1)'>
+    <g transform='translate(0.75, -1)'>
```

Comment:
these need to be perfectly centered for the rotation to work - maybe you have some ideas on how that can work better than the translating? Perhaps we should try to export them perfectly or something and re-path them?